specification/configuration.md

```diff
@@ -130,6 +136,32 @@ beyond the OpenTelemetry specification exist.
   - Distribution MUST default to `otlp`
   - Distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
 
+### Real User Monitoring Libraries
+
+Real User Monitoring (RUM) instrumentation libraries cannot use environment variables for configuration.
+Instead, they MUST expose a `SplunkRum` object/class/namespace (depending on the language used) that allows
+setting the following properties (in a language-specific way: Java may use builders, Swift can use named
+parameters with default values, JavaScript can use objects, etc.):
+
+| Configuration property (default value) | Description                                                                                                                                                                                                 | Required |
+| -------------------------------------- | -----------                                                                                                                                                                                                 | -------- |
+| `realm` ()                             | Splunk realm, e.g. `us0`, `us1`. If set, value of `beaconUrl` will be automatically computed based on this. [1] [2]                                                                                         | Yes      |
+| `beaconUrl` ()                         | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. If both `realm` and `beaconUrl` are set, `beaconUrl` takes precedence. [1] [2]                                                        | Yes      |
```

Comment:
We seem to use `endpoint` in most places (e.g. `OTEL_EXPORTER_JAEGER_ENDPOINT` or see receivers/exporters in collector), can this be named `beaconEndpoint` for consistency?

```suggestion
| `beaconEndpoint` ()                         | RUM beacon endpoint, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. If both `realm` and `beaconEndpoint` are set, `beaconEndpoint` takes precedence. [1] [2]                                                        | Yes      |
```

Reply:
+1 for this :)